test/http-error.ts

```diff
@@ -1,18 +1,18 @@
 import test from 'ava';
 import {HTTPError} from '../source/index.js';
 
+type Mutable<T> = {
+	-readonly[P in keyof T]: T[P]
+};
+
 function createFakeResponse({status, statusText}: {status?: number; statusText?: string}): Response {
 	// Start with a realistic fetch Response.
-	const response = {...new Response()};
+	const response: Partial<Mutable<Response>> = {...new Response()};
 
-	// Assign these values because the Response() constructor doesn't
```

Comment:
There are 2 problems here:

1. it is `readonly`
2. it doesn't accept `undefined`